### PR TITLE
[MEN-29] Initial project configuration

### PR DESCRIPTION
**Pull request by gabrielmethot**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+		"dbaeumer.vscode-eslint",
```
Comment (laurentlp) [inline .vscode/extensions.json]: Weird! Shouldn't this file be automatically formatted?

File: next-env.d.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
```
Comment (laurentlp) [inline next-env.d.ts:2]: Any reasons why those lines are commented?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/menmos/menmos-web.git"
+  },
+  "bugs": {
+    "url": "https://github.com/menmos/menmos-web/issues"
+  },
+  "scripts": {
+    "dev": "next dev",
```
Comment (laurentlp) [inline package.json:13]: `dev` is the equivalent of watch?

File: public/favicon.ico [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (laurentlp) [inline public/favicon.ico]: Is this normal?

**Review by laurentlp — APPROVED** [review]
Small comments other than that LGTM!! Great job!!!! :100:

File: .vscode/extensions.json (hunk 1 above)
Comment (gabrielmethot) [reply .vscode/extensions.json]: It looks weird in Github but is formatted properly on my machine. Can you clone the repo and tell me if the indentation is still messed up?

File: package.json (hunk 3 above)
Comment (gabrielmethot) [reply package.json:13]: Kind of, it runs the application in [development mode](https://nextjs.org/docs/api-reference/cli#development). Some key differences:
- Hot reloads your changes
- Type errors do not the application
- `getStaticProps()` is called on every request

File: public/favicon.ico (hunk 4 above)
Comment (gabrielmethot) [reply public/favicon.ico]: Nope, I accidently ran StyleLint on non-CSS files and it messed it up :')

File: next-env.d.ts (hunk 2 above)
Comment (gabrielmethot) [reply next-env.d.ts:2]: https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html

File: .vscode/extensions.json (hunk 1 above)
Comment (laurentlp) [reply .vscode/extensions.json]: Looks fine on my end too! @Github plissss!

**Comment by laurentlp** [comment]
Euhm...

Screenshot from 2021-04-02 16-38-44

Can you tell me which version of node and npm you're using?

_(I have no issue with NodeJS >=15)_

**Comment by gabrielmethot** [comment]
> Euhm...
> 
> Screenshot from 2021-04-02 16-38-44
> 
> Can you tell me which version of node and npm you're using?
> 
> _(I have no issue with NodeJS >=15)_

My bad, I installed dependencies using a Node.js version other than the one specified in the `.nvmrc` file. I regenerated the lock file using Node.js version 14.16.0, which comes with NPM version 6.14.11 by default. Tell me if that fixes it and if that's ok with you.

**Review by dalloriam — APPROVED** [review]

**Pull request merged**